BMIService.kt

```diff
@@ -0,0 +1,17 @@
+package com.myfitnesstracker.service
+
+import BMI_DAO
```

Comment:
A few naming conventions are not followed here:

1) Class names should be UpperCamelCase, not ALL CAPS.
2) We typically don't use underscores (_) in class names.
3) The class name ends with DAO, yet it is in a service package.
4) If it is in a DAO package, you can drop the "DAO" from the name... thus, a more appropriate name for the class is simply BodyMassIndex.